source/_components/switch.pilight.markdown

```diff
@@ -55,6 +55,12 @@ Variables for the different codes:
 - **'off'** (*Optional*): `1` or `0`
 - **'on'** (*Optional*): `1` or `0`
 
+Besides that the he `on_code_receive` and `off_code_receive` have the following special option:
+
+- **echo** (*Optional*) `true` if the on-/off-code should be sent if the given code was received.
+
+This is useful if you have paired your sender directly with the receiver to prevent sending the signal twice.
```

Comment:
It's not clear when looking at the output that this paragraph is only referring to the `echo` option. Perhaps the paragraph below should move before the additions and/or `echo` should be in a sub-section?